account_statement_base/views/account_bank_statement_line.xml

```diff
@@ -0,0 +1,158 @@
+<?xml version="1.0" ?>
+<!--
+  Copyright 2022-2023 CreuBlanca
+  Copyright 2023 Akretion France (http://www.akretion.com/)
+  @author: Alexis de Lattre <alexis.delattre@akretion.com>
+  Licence LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0).
+-->
+<odoo>
+
+<record id="account_bank_statement_line_form" model="ir.ui.view">
+    <field name="model">account.bank.statement.line</field>
+    <field name="arch" type="xml">
+        <form>
+            <group name="main">
+                <field name="company_id" invisible="1" />
+                <field name="currency_id" invisible="1" />
+                <field name="country_code" invisible="1" />
+                <field name="state" invisible="1" />
+                <field name="suitable_journal_ids" invisible="1" />
+                <field name="date" />
+                <field name="payment_ref" required="1" />
+                <field name="partner_id" />
+                <field name="amount" />
+                <field
+                        name="foreign_currency_id"
+                        domain="[('id', '!=', currency_id)]"
+                        groups="base.group_multi_currency"
+                    />
+                <field name="amount_currency" groups="base.group_multi_currency" />
+                <field
+                        name="partner_bank_id"
+                        invisible="not context.get('add_bank_statement_line')"
+                    />
+                <field
+                        name="journal_id"
+                        invisible="context.get('default_journal_id')"
+                    />
+                <field
+                        name="move_id"
+                        invisible="not context.get('add_bank_statement_line')"
+                    />
+                <field
+                        name="statement_id"
+                        domain="[('journal_id', '=', journal_id)]"
+                        invisible="not context.get('account_bank_statement_line_main_view')"
+                    />
+            </group>
+            <notebook invisible="not context.get('add_bank_statement_line')">
+                <page name="narration" string="Notes">
+                    <field name="narration" nolabel="1" colspan="2" />
+                </page>
+                <page name="technical" string="Technical Information">
+                    <group name="tech-fields">
+                        <field name="transaction_type" />
+                        <field name="partner_name" />
+                        <field name="account_number" />
+                        <field name="is_reconciled" />
+                        <field name="payment_ids" widget="many2many_tags" />
+                    </group>
+                </page>
+            </notebook>
+            <footer invisible="not context.get('add_bank_statement_line')">
+                <button name="save" type="object" string="Save" class="btn-primary" />
```

Comment:
as there is no save method on the object, it seems you should use a Javascript "special" attribute, something like
```
                        <button string="Confirm" class="btn-primary" special="save" data-hotkey="v"/>
                        <button string="Cancel" class="btn-secondary" special="cancel" data-hotkey="z"/>
```

see failing test https://github.com/OCA/account-reconcile/actions/runs/3952659874/jobs/6768008598#step:8:64

Reply:
on a second thought, as the account.bank.statement.line form is not meant to be edited in a bank statement anymore, shouldn't you just remove this footer?

Reply:
You're right. I did it yesterday on my laptop, but I forgot to push it to github before going to bed, sorry.